sklearn/datasets/_samples_generator.py

```diff
@@ -154,6 +154,11 @@ def make_classification(
 
     y : ndarray of shape (n_samples,)
         The integer labels for class membership of each sample.
+    
```

Comment:
To fix the [linting issues](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=39361&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=8a54543f-0728-5134-6642-bedd98e03dd0):

```suggestion

```